integration-test/features/services.feature

```diff
@@ -2,18 +2,54 @@ Feature: All about Services
 
   Background:
     Given GPS running
-
+    
+  Scenario: create a new service
+    Given the body of the request for a service with id "service-2"
```

Comment:
could be useful re-write given as made [here](https://pagopa.atlassian.net/wiki/spaces/PAG/pages/540639233/Test+cases#GetFeeByPsp)